python/tvm/relay/backend/contrib/ethosu/tir_to_cs_translator.py

```diff
@@ -877,7 +877,7 @@ def _create_npu_activation(serial_activation: spec.SerialActivation) -> vapi.Npu
         return None
     op_map = {
         "CLIP": vapi.NpuActivationOp.NONE_OR_RELU,
-        "TANH": vapi.NpuActivationOp.TABLE_LOOKUP,
+        "TANH": vapi.NpuActivationOp.TANH,
```

Comment:
This change would make it always use the NPU's builtin tanh function instead of the calculated lookup table values, making it to not match TFLite reference kernels and turning all tanh LUT value calculation into dead code.

Reply:
[PR](https://github.com/apache/tvm/pull/16680) was merged and the code has been updated.